ci/test-container.sh

```diff
@@ -56,19 +56,19 @@ versionid=$(. /usr/lib/os-release && echo $VERSION_ID)
 
 # Test overrides
 case $versionid in
-  38)
+  40)
     url_suffix=2.16.2/2.fc39/x86_64/ignition-2.16.2-2.fc39.x86_64.rpm
     # 2.15.0-3
     koji_url="https://koji.fedoraproject.org/koji/buildinfo?buildID=2158585"
-    koji_kernel_url="https://koji.fedoraproject.org/koji/buildinfo?buildID=2174317"
-    kver=6.2.8
+    koji_kernel_url=https://koji.fedoraproject.org/koji/buildinfo?buildID=2294111
```

Comment:
```suggestion
    koji_kernel_url=https://koji.fedoraproject.org/koji/buildinfo?buildID=2435097
```

Reply:
We are testing for the version ID, so either we change that or we use a package from the same version(f40)